apps/marketplace/pages/DownloadReports/DownloadReports.js

```diff
@@ -0,0 +1,213 @@
+import React, { Component } from 'react'
+import AUbutton from '@gov.au/buttons/lib/js/react.js'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+
+import PropTypes from 'prop-types'
+import formProps from 'shared/form/formPropsSelector'
+import RadioList from 'shared/form/RadioList'
+import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
+import ErrorAlert from 'marketplace/components/Alerts/ErrorAlert'
+import { Form, actions } from 'react-redux-form'
+import { AUcheckbox } from '@gov.au/control-input'
+import { Redirect } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { hasPermission } from 'marketplace/components/helpers'
+import { rootPath } from 'marketplace/routes'
+import { required, validDate, startDateIsBeforeEndDate } from 'marketplace/components/validators'
+
+import styles from '../../main.scss'
```

Comment:
Should these styles be in `DownloadReports.scss`?

Reply:
reusing common styles for spacing